packages/api/src/external/hie/carequality-analytics.ts

```diff
@@ -178,16 +182,56 @@ export function getOutboundDocQuerySuccessFailureCount(response: OutboundDocumen
 
 export function getOutboundDocRetrievalSuccessFailureCount(
   response: OutboundDocumentRetrievalResp[]
-): { successCount: number; failureCount: number } {
+): {
+  totalCount: number;
+  successCount: number;
+  failureCount: number;
+  noDocumentFoundCount: number;
+  schemaErrorCount: number;
+  httpErrorCount: number;
+  registryErrorCount: number;
+} {
   let successCount = 0;
```

Comment:
i really dont like these blocks of `let` vars. Can't we define an object with these as attributes and also define a type for these so the func return is cleaner?